src/datasets/bigquery/clients_last_seen/reference.md

```diff
@@ -0,0 +1,73 @@
+# Clients Last Seen Reference
+
+<!-- toc -->
+
+# Introduction
+
+{{#include ./intro.md}}
+
+# Data Reference
+
+## Example Queries
+
+#### Compute DAU for non-windows clients for the last week
+
+```sql
+SELECT
+    submission_date,
+    CURRENT_TIMESTAMP() AS generate_time,
```

Comment:
Why include a generated time in these examples? If this should stay, should it be `generated_timestamp` to be consistent?

Reply:
we consistently call it `generated_time` in bigquery-etl, so I think we should stick with that

I think if we use the field in bigquery-etl that we should use it in examples, so that people get used to seeing and including it as a general good practice. that said, I think we might want to drop the column in general as per mozilla/bigquery-etl#38

Reply:
removing the column